common/reviews/api/rush-lib.api.md

```diff
@@ -402,6 +428,7 @@ export class RushConfigurationProject {
     readonly downstreamDependencyProjects: string[];
     // @beta
     readonly isMainProject: boolean;
+    readonly localDependencyProjects: Map<string, RushConfigurationProject>;
```

Comment:
Can this be a simple list? The map key is a member of the map value.

Reply:
Sure, only reason I kept it this way was for quick lookup, but a set can be created by mapping the values so that's not a big deal